src/CoreWlan/CWKeychain.cs

```diff
@@ -27,6 +27,7 @@ public static partial class CWKeychain {
 
 #if NET
 		[SupportedOSPlatform ("macos10.9")]
+		[SupportedOSPlatform ("maccatalyst15.0")]
```

Comment:
Notes for reviewers, this is the owning class:

```
	[MacCatalyst (15,0)]
	public static partial class CWKeychain
```

Reply:
👀